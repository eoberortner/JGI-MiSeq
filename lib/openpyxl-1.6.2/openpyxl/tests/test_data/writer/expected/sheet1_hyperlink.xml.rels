<?xml version='1.0' encoding='UTF-8'?>
<Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships">
    <Relationship Id="rId1" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink" Target="http://test.com/" TargetMode="External"/>
    <Relationship Id="rId2" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink" Target="http://test2.com/" TargetMode="External"/>
</Relationships>

