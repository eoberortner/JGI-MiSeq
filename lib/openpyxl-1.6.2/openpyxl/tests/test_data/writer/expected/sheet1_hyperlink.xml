<?xml version='1.0' encoding='UTF-8'?>
<worksheet xml:space="preserve" xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships">
    <sheetPr>
        <outlinePr summaryBelow="1" summaryRight="1" />
    </sheetPr>
    <dimension ref="A1:A1" />
    <sheetViews>
        <sheetView workbookViewId="0">
            <selection activeCell="A1" sqref="A1" />
        </sheetView>
    </sheetViews>
    <sheetFormatPr defaultRowHeight="15" />
    <cols>
        <col collapsed="0" max="1" min="1" width="9.10" />
    </cols>
    <sheetData>
        <row r="1" spans="1:1">
            <c r="A1" t="s">
                <v>0</v>
            </c>
        </row>
    </sheetData>
    <hyperlinks>
        <hyperlink display="http://test.com" ref="A1" r:id='rId1'/>
    </hyperlinks>
</worksheet>

